### DISCUSSION TITLE
Icon Request: fa-scout-sign

### DISCUSSION THREAD

**Discussion by elbel86** [FortAwesome/Font-Awesome#21200] [Icon Requests]
The three finger scout sign of the Boy Scouts of America.  We already have the cub scout sign fa-hand-peace-o (flip it for correctness).
![hand-sign](https://cloud.githubusercontent.com/assets/18019642/13973572/6b74d476-f07a-11e5-80f2-8bdcbe776e7e.gif)

**Comment by Jesse97** [comment]
Being a Dutch member of Scouting I'd like to point out that this sign is used internationally.

+1

**Comment by nklein1** [comment]
+1

**Comment by RileyStarlight** [comment]
Not only in Boy Scouts of America, also as Scouts here in Europe this is used too.
+1

**Comment by jesusignazio** [comment]
+1

**Comment by HeapsGoodTom** [comment]
+1 - Used globally, as laid out by the founder, Sir Robert Baden-Powell.

**Comment by ryanknutson** [comment]
I'd like to add the [fleur de lis](http://www.nesa.org/logo/fleur-de-lis.jpg) to this as well

**Comment by christaylortf** [comment]
+1

**Comment by gitChang** [comment]
+1 really need this.

**Comment by jkgarner** [comment]
I concur about the scout Fleur-de-Lis as an icon. It is used world-wide
![fleur-de-lis](https://user-images.githubusercontent.com/3368943/169033852-3dde1084-51e3-4095-9bf0-f9f0e2fee596.svg)

.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
